src/components/victory-candlestick/victory-candlestick.js

```diff
@@ -0,0 +1,433 @@
+      PropTypes.string,
+      PropTypes.arrayOf(PropTypes.string)
+    ]),
+    /**
+     * The y prop specifies how to access the Y value of each data point.
+     * If given as a function, it will be run on each data point, and returned value will be used.
+     * If given as an integer, it will be used as an array index for array-type data points.
+     * If given as a string, it will be used as a property key for object-type data points.
+     * If given as an array of strings, or a string containing dots or brackets,
+     * it will be used as a nested object property path (for details see Lodash docs for _.get).
+     * If `null` or `undefined`, the data value will be used as is (identity function/pass-through).
+     * @examples 0, 'y', 'y.value.nested.1.thing', 'y[2].also.nested', null, d => Math.sin(d)
+     */
+    y: PropTypes.oneOfType([
+      PropTypes.func,
+      CustomPropTypes.allOfType([CustomPropTypes.integer, CustomPropTypes.nonNegative]),
+      PropTypes.string,
+      PropTypes.arrayOf(PropTypes.string)
+    ])
+  };
+
+  static defaultProps = {
+    events: {},
+    height: 300,
+    padding: 50,
+    samples: 50,
+    scale: "linear",
+    size: 3,
+    standalone: true,
+    symbol: "circle",
+    width: 450,
+    candleColors: {
+      positive: "green",
+      negative: "red"
+    },
+    x: "x",
+    y: "y",
+    dataComponent: <Candle/>,
+    labelComponent: <VictoryLabel/>
+  };
+
+  static getDomain = Domain.getDomain.bind(Domain);
+  static getData = Data.getData.bind(Data);
+
+  constructor() {
+    super();
+    this.state = {};
+    this.getEvents = Helpers.getEvents.bind(this);
+    this.getEventState = Helpers.getEventState.bind(this);
+  }
+
+  getDataStyles(data, style) {
+    const stylesFromData = omit(data, [
+      "x", "y", "z", "size", "symbol", "name", "label"
+    ]);
+    const baseDataStyle = defaults({}, stylesFromData, style);
+    return Helpers.evaluateStyle(baseDataStyle, data);
+  }
+
+  getLabelText(props, datum, index) {
+    const propsLabel = Array.isArray(props.labels) ?
+      props.labels[index] : Helpers.evaluateProp(props.labels, datum);
+    return datum.label || propsLabel;
+  }
+
+  getLabelStyle(labelStyle, dataProps) {
+    const { datum, size, style } = dataProps;
+    const matchedStyle = pick(style, ["opacity", "fill"]);
+    const padding = labelStyle.padding || size * 0.25;
+    const baseLabelStyle = defaults({}, labelStyle, matchedStyle, {padding});
+    return Helpers.evaluateStyle(baseLabelStyle, datum);
+  }
+
+  renderData(props, calculatedProps, style) {
+    const dataEvents = this.getEvents(props.events.data, "data");
+    const labelEvents = this.getEvents(props.events.labels, "labels");
+    const { scale, data } = calculatedProps;
+    return data.map((datum, index) => {
+      // const x = scale.x(datum.x);
+      // const wickY1 = scale.y(datum.high);
+      // const wickY2 = scale.y(datum.low);
+      const candleX = datum.x;
```

Comment:
these position type values like `wickX` should be scaled before being passed to the candle component.